Traceback (most recent call last):
  File "/project/p_gnn001/code/tsp/tsp/train2.py", line 21, in <module>
    from gnngls import models, datasets, algorithms
  File "/project/p_gnn001/code/tsp/tsp/gnngls/datasets.py", line 212, in <module>
    directed_string_graph(complete_graph)
  File "/project/p_gnn001/code/tsp/tsp/gnngls/datasets.py", line 91, in directed_string_graph
    ss.append((edge_id[(i, j)], edge_id[(i, v)]))
               ~~~~~~~^^^^^^^^
KeyError: (0, 1)
